XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs

```diff
@@ -642,9 +642,23 @@ public override bool EnableNext()
         public Pool_patch Patch { private get; set; }
         public Pool_update PoolUpdate { private get; set; }
 
+        public List<AsyncAction> GetUnwindChangesActions()

```

Comment:
Everytime this function is called (when moving away from the prechecks page and also on cancelling the wizard), it creates the unwind actions and the actions that are created on leaving the page are shown in the Events page before they are actually executed. There will also be duplicates when moving back to the prechecks page and then to the next page again (this is possible in the RPU wizard only)